README.md

```diff
@@ -58,11 +58,16 @@ to your plugin manager, e.g. after `call plug#end()` with vim-plug.)
 ### Advanced setup
 
 The author likes the following, which uses different modes based on if your
-laptop runs on battery (for Linux):
+laptop runs on battery (for MacOS or Linux):
 
 ```vim
 function! MyOnBattery()
-  return readfile('/sys/class/power_supply/AC/online') == ['0']
+  if has('macunix')
+    return match(system('pmset -g batt'),"Now drawing from 'Battery Power'") != -1
```

Comment:
Just for reference: I've found https://github.com/imwally/battstat/blob/master/battstat#L38 - which just checks for "Battery Power" (with word boundaries).